flink-runtime/src/main/java/org/apache/flink/runtime/executiongraph/ExecutionGraph.java

```diff
@@ -49,7 +49,7 @@
 import org.apache.flink.runtime.concurrent.ScheduledExecutorServiceAdapter;
 import org.apache.flink.runtime.entrypoint.ClusterEntryPointExceptionUtils;
 import org.apache.flink.runtime.execution.ExecutionState;
-import org.apache.flink.runtime.executiongraph.failover.FailoverStrategy;
+import org.apache.flink.runtime.executiongraph.failover.flip1.FailoverStrategy;
```

Comment:
I think this import is not really needed if we adjust the JavaDocs.

Reply:
true. I think we can remove the JavaDoc about Failover because it is outdated.

Reply:
done.